tests/tags/test_lorem.py

```diff
@@ -0,0 +1,166 @@
+from django.utils.lorem_ipsum import COMMON_P, WORDS
+
+
+def test_lorem_words(render_output):
+    template = "{% lorem 3 w %}"
+    output = render_output(
+        template=template,
+        context={},
+    )
+    assert "lorem ipsum dolor" == output
+
+
+def test_lorem_random(render_output):
+    template = "{% lorem 3 w random %}"
+    output = render_output(template=template, context={})
+
+    words = output.split(" ")
+    assert len(words) == 3
+    for word in words:
+        assert word in WORDS
+
+
+def test_lorem_default(assert_render):
+    template = "{% lorem %}"
+    assert_render(
+        template=template,
+        context={},
+        expected=COMMON_P,
+    )
+
+
+def test_lorem_multiple_paragraphs(render_output):
+    template = "{% lorem 2 p %}"
+    output = render_output(template=template, context={})
+    assert output.count("<p>") == 2
+
+
+def test_lorem_incorrect_count(render_output):
+    template = "{% lorem two p %}"
+    output = render_output(template=template, context={})
+    assert output.count("<p>") == 1
+
+
+def test_lorem_syntax_error(assert_parse_error):
+    template = "{% lorem 1 2 3 4 %}"
+
+    django_message = "Incorrect format for 'lorem' tag"
+
+    rusty_message = """\
+  × Incorrect format for 'lorem' tag
+   ╭────
+ 1 │ {% lorem 1 2 3 4 %}
+   ·            ┬
+   ·            ╰── here
+   ╰────
```

Comment:
I think we can provide better error messages than Django. For example, I think this can be something like:

```suggestion
  × Incorrect format for 'lorem' tag: 'count' argument was provided more than once
   ╭────
 1 │ {% lorem 1 2 3 4 %}
   ·          ┬ ┬
   ·          │ ╰── second 'count'
   ·          ╰── first 'count'
   ╰────
  help: Try removing the second 'count'
```